src/react_native/safe_area.cljs

```diff
@@ -10,7 +10,8 @@
 
 (defn- get-static-bottom
   []
-  (oops/oget StaticSafeAreaInsets "safeAreaInsetsBottom"))
+  (some-> StaticSafeAreaInsets
+          (oops/oget "safeAreaInsetsBottom")))
```

Comment:
StaticSafeAreaInsets is not defined while executing component-tests, so they were failing now that I'm using this function in a component that is tested (documentation-drawer).

This function now returns `nil` when the value is not defined, however, the `get-bottom` function (below) returns zero when this value is not defined.